### PR TITLE
JPO - Step 5 (summary) style adjustments

### PR DESCRIPTION
**Pull request by ashleighaxios**
* fixed icon color and size to match Calypso standard sizes
* adjusted mobile wrapping
* adjusted wording
* added links to the second column
* aligned text across two columns

### CODE REVIEW

File: client/signup/steps/jpo-summary/index.jsx [hunk 1]
```diff
@@ -172,7 +172,7 @@ class JPOSummaryStep extends React.Component {
 				<div>
 					<Button primary onClick={ this.completeOnboarding }>
 						{
-							translate( 'Complete Onboarding' )
+							translate( 'Visit your site' )
```
Comment (a8ci18n) [inline client/signup/steps/jpo-summary/index.jsx]: Hi! I've found a possible matching string that has already been translated **38** times:
`translate( 'Visit Your Site' )` *ES Score: 11*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/signup/steps/jpo-summary/index.jsx [hunk 2]
```diff
@@ -159,11 +159,11 @@ class JPOSummaryStep extends React.Component {
...
 											: <li><a href={ `${ pathPrefix }jpo-contact-form` }>{ translate( 'Contact Us Form' ) }</a></li>
 									}
-									{ ! connectionToJetpackComplete ? <li>{ translate( 'Connection to Jetpack' ) }</li> : null }
-									<li>{ translate( 'Choose a Theme' ) }</li>
-									<li>{ translate( 'Add a Site Address' ) }</li>
-									<li>{ translate( 'Add a Store' ) }</li>
-									<li>{ translate( 'Start a Blog' ) }</li>
+									{ ! connectionToJetpackComplete ? <li>{ translate( 'Jetpack Connection' ) }</li> : null }
+									<li><a href="#">{ translate( 'Choose a Theme' ) }</a></li>
+									<li><a href="#">{ translate( 'Add a Site Address' ) }</a></li>
```
Comment (a8ci18n) [inline client/signup/steps/jpo-summary/index.jsx]: :disappointed: 1 existing translations will be lost with this change.
If the string must change, the following option has already been translated **85** times:
`translate( 'Site Address' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/signup/steps/jpo-summary/index.jsx [hunk 3]
```diff
@@ -107,35 +107,35 @@ class JPOSummaryStep extends React.Component {
...
 									{
 										jpoHomepage
-											? <li>{ checkMark } { translate( 'Type of Homepage' ) }</li>
+											? <li>{ checkMark } <span>{ translate( 'Type of Homepage' ) }</span></li>
 											: null
 									}
 									{
 										jpoContactForm
-											? <li>{ checkMark } { translate( 'Contact Us Form' ) }</li>
+											? <li>{ checkMark } <span>{ translate( 'Contact Us Form' ) }</span></li>
```
Comment (a8ci18n) [inline client/signup/steps/jpo-summary/index.jsx]: Hi! I've found a possible matching string that has already been translated **31** times:
`translate( 'contact us' )` *ES Score: 7*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/jpo-final-step-cleanup)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

**Comment by eliorivero** [comment]
Looks good!

**Pull request merged**